zcash_client_backend/src/data_api/chain.rs

```diff
@@ -198,44 +186,6 @@ where
 ///
 /// Scanned blocks are required to be height-sequential. If a block is missing from the
 /// cache, an error will be returned with kind [`ChainInvalid::BlockHeightDiscontinuity`].
-///
-/// # Examples
```

Comment:
The rest of them I fixed, but this one is so trivial that I figured it could go; it actually doesn't give you any more information than the type signature.